packages/core/src/components/forms/_control-group.scss

```diff
@@ -30,34 +31,70 @@ child must be marked individually as `.pt-large` for uniform large appearance.
+    <button class="pt-button">Button</button>
+  </div>
+  <div class="pt-control-group">
+    <button class="pt-button">Button</button>
+    <input type="text" class="pt-input pt-intent-success" placeholder="I am a text input..." />
+  </div>
+  <div class="pt-control-group">
+    <input type="text" class="pt-input pt-intent-warning" placeholder="I am a text input..." />
+    <button class="pt-button pt-intent-success" disabled>Button</button>
+  </div>
+  <div class="pt-control-group">
+    <button class="pt-button pt-intent-warning" disabled>Button</button>
+    <button class="pt-button pt-intent-success">Button</button>
+  </div>
+  <div class="pt-control-group">
+    <button class="pt-button pt-intent-warning">Button</button>
+    <button class="pt-button pt-intent-success">Button</button>
+  </div>
+  <div class="pt-control-group">
+    <button class="pt-button pt-intent-warning">Button</button>
+    <input type="text" class="pt-input pt-intent-success" placeholder="I am a text input..." />
   </div>
   <div class="pt-control-group">
-    <div class="pt-select">
-      <select>
-        <option selected>Filter...</option>
-        <option value="1">Issues</option>
-        <option value="2">Requests</option>
-        <option value="3">Projects</option>
-      </select>
-    </div>
     <div class="pt-input-group">
-      <span class="pt-icon pt-icon-search"></span>
-      <input type="text" class="pt-input" value="from:ggray to:allorca" />
+      <span class="pt-icon pt-icon-people"></span>
+      <input type="text" class="pt-input pt-intent-success" placeholder="I am a text input..." style="padding-right:94px" />
+      <div class="pt-input-action">
+        <button class="pt-button pt-minimal pt-intent-primary">
+          can view<span class="pt-icon-standard pt-icon-caret-down pt-align-right"></span>
+        </button>
+      </div>
     </div>
+    <button class="pt-button pt-intent-warning">Button</button>
+    <button class="pt-button pt-intent-success">Button</button>
+    <button class="pt-button">Button</button>
   </div>
   <div class="pt-control-group">
     <div class="pt-input-group">
       <span class="pt-icon pt-icon-people"></span>
-      <input type="text" class="pt-input" placeholder="Find collaborators..." style="padding-right:94px" />
+      <input type="text" class="pt-input pt-intent-success" placeholder="I am a text input..." style="padding-right:94px" />
       <div class="pt-input-action">
         <button class="pt-button pt-minimal pt-intent-primary">
           can view<span class="pt-icon-standard pt-icon-caret-down pt-align-right"></span>
         </button>
       </div>
     </div>
-    <button class="pt-button pt-intent-primary">Add</button>
+    <div class="pt-button-group">
+      <button class="pt-button pt-intent-warning">Button</button>
+      <button class="pt-button pt-intent-success">Button</button>
+      <button class="pt-button">Button</button>
+    </div>
```

Comment:
@cmslewis are we planning to ship all these example changes? on one hand, they're useful for testing, but on the other they're somewhat unsightly and are usages that we may not want to promote so visibly.